src/bp/core/services/ghost/memory-cache.ts

```diff
@@ -14,9 +15,14 @@ export default class MemoryObjectCache implements ObjectCache {
 
   constructor(@inject(TYPES.FileCacheInvalidator) private cacheInvalidator: CacheInvalidators.FileChangedInvalidator) {
     this.cache = LRU({
-      // For now we cache up to 5000 elements, whatever the size
-      // We will probably want to assign different length to various element types in the future
-      max: 5000
+      max: asBytes(process.core_env.BP_MAX_MEMORY_CACHE_SIZE || '1gb'),
+      length: (n, key) => {
+        if (key.startsWith('buffer::')) {
+          return n.length
+        }
+
+        return 500 // Assuming 500 bytes per objects, this is kind of random
```

Comment:
Assuming a non empty bot, I looked it up quickly and content & most read config obj are more around 1kb. But i guess a lower bound is safer ?

Reply:
fixed!